### DISCUSSION TITLE
Quarkus extension for AWS SQS

### DISCUSSION THREAD

**Discussion by vivektk84** [quarkusio/quarkus#40766] [Community]
Now that SmallRye supports AWS SQS(https://github.com/smallrye/smallrye-reactive-messaging/pull/2500), is there a plan to bring it as a Quarkus extension like what we have currently for Kafka?

**Comment by adampoplawski** [comment]
Hello
Any plans to support it, or is low prio?

**Reply by mschorsch** [reply]
https://quarkus.io/extensions/io.quarkiverse.amazonservices/quarkus-amazon-sqs/
